src/components/sectionList/translation/TranslatableFields.tsx

```diff
@@ -0,0 +1,77 @@
+import i18n from '@dhis2/d2-i18n' // Importing i18n here
+import {
+    InputField,
+    InputFieldFF,
+    TextAreaField,
+    TextAreaFieldFF,
+} from '@dhis2/ui'
+import React from 'react'
+import { Field } from 'react-final-form'
+import { getTranslatedProperty } from '../../../lib'
+import { camelCaseToConstantCase } from '../../../lib/utils/caseTransformers'
+
+interface TranslatableFieldsProps {
+    translatableFields: string[]
+    baseReferenceValues: Record<string, string>
+    selectedLocale?: { locale: string } | undefined
+    style: any
```

Comment:
When we pass style I would've expected that to be the css-in-js object, eg "{ margin: "0px" }". What we pass here is actually just an object of css-names, so the type would be `Record<string,string>`. 

But I think we can just remove this prop all together and just import the styles directly in this file instead.